src/lib/homestore_backend/heap_chunk_selector.h

```diff
@@ -70,6 +70,7 @@ class HeapChunkSelector : public homestore::ChunkSelector {
     std::unordered_map< chunk_num_t, csharedChunk > m_chunks;
 
     VChunkDefragHeap m_defrag_heap;
+    std::unordered_set< chunk_num_t > m_selected_chunks;
```

Comment:
m_selected_chunks means chunk with  opening shard?  I do not see anyway to put element into it.

Reply:
at  heap_chunk_selector.cpp line 178 , we should m_selected_chunks.emplace(chunkID), not erase, this is a mistake, will fix it.